src/ui/server/helpers/whole-number-validation/index.ts

```diff
@@ -0,0 +1,27 @@
+import { RequestBody } from '../../../types';
+import generateValidationErrors from '../validation';
+import { isNumber, numberHasDecimal, isNumberBelowMinimum } from '../number';
+
+/**
+ * validates if field is whole number and above 0
+ * returns validation error if is not a number, has a decimal place or special characters or is below 0
+ * @param {RequestBody} responseBody
+ * @param {object} errors
+ * @param {object} errorField error field to get the error message
+ * @param {string} field fieldId of the field being checked
+ * @returns {object} errors
+ */
+const wholeNumberValidation = (responseBody: RequestBody, errors: object, errorField: object, field: string) => {
+  const hasDecimal = numberHasDecimal(Number(responseBody[field]));
+  const isBelowMinimum = isNumberBelowMinimum(Number(responseBody[field]), 0);
+
+  if (!isNumber(responseBody[field]) || hasDecimal || isBelowMinimum) {
+    const errorMessage = errorField[field].INCORRECT_FORMAT;
+
+    return generateValidationErrors(field, errorMessage, errors);
+  }
+
+  return errors;
+};
+
```

Comment:
Nice - i'm wondering if it's worth having a new number helper `isWholeNumber` which does the checks (line 15 to 18), and then we consume the new helper in this validation file. What do you think?

Because then we'd have the ability to check for whole numbers without having to pass validation errors. Gives more flexibility

Reply:
can we make a tech debt for this please?

Reply:
https://ukef-dtfs.atlassian.net/browse/EMS-891